aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -69,6 +69,34 @@ void geqrf_batched_cublas(const Tensor& input, const Tensor& tau) {
   });
 }
 
+template <typename scalar_t>
+static void apply_lu_solve_batched_cublas(const Tensor& b, const Tensor& lu, const Tensor& pivots) {
+  cublasOperation_t trans = CUBLAS_OP_N;
+
+  auto pivots_data = pivots.data_ptr<int>();
+  auto batch_size = cuda_int_cast(batchCount(lu), "batch_size");;
+  auto m = cuda_int_cast(lu.size(-2), "m");
+  auto nrhs = cuda_int_cast(b.size(-1), "nrhs");
+  auto lda = cuda_int_cast(std::max<int>(1, m), "lda");
+  Tensor infos_getrs = at::zeros({std::max<int>(1, batch_size)}, lu.options().dtype(kInt).device(kCPU));
+  auto infos_getrs_data = infos_getrs.data_ptr<int>();
```

Comment:
cuBLAS's getrs takes a single integer in host memory and indicates whether passed-in arguments are correct. We don't need Tensor for that.
```suggestion
  int info = 0;
```